test/helpers/IntegrationHelper.js

```diff
@@ -836,22 +840,24 @@ global.integration = function (definitions) {
                 this.player1.moveAllNonBaseZonesToRemoved();
                 this.player2.moveAllNonBaseZonesToRemoved();
 
-                // Resources
-                this.player1.setResourceCards(options.player1.resources, ['removed from game']);
-                this.player2.setResourceCards(options.player2.resources, ['removed from game']);
-
-                // Arenas
-                this.player1.setGroundArenaUnits(options.player1.groundArena, ['removed from game']);
-                this.player2.setGroundArenaUnits(options.player2.groundArena, ['removed from game']);
-                this.player1.setSpaceArenaUnits(options.player1.spaceArena, ['removed from game']);
-                this.player2.setSpaceArenaUnits(options.player2.spaceArena, ['removed from game']);
-
-                // Hand + discard
-                this.player1.setHand(options.player1.hand, ['removed from game']);
-                this.player2.setHand(options.player2.hand, ['removed from game']);
-                this.player1.setDiscard(options.player1.discard, ['removed from game']);
-                this.player2.setDiscard(options.player2.discard, ['removed from game']);
 
+                if (options.phase !== 'setup') {
```

Comment:
This is a good start. One thing we need to consider though is that people might get confused and still try to set some of the configuration settings not realizing that they don't apply in setup phase tests. So it would be good to have some feedback mechanism for letting them know. 

Probably ought to be some kind of parameter checks where we throw an error if they've done something that doesn't apply in this type of test. This would apply not only for putting in cards in arenas like in this if block, but also if they try to set leader status deployed or base damage in the below settings.

I've honestly been meaning for a while to add some validation checks on the `options.playerX` config objects, mostly to catch if someone has a typo in their config and accidentally puts `groundAerna` or something because that would just silently fail. Maybe this is a good chance to add a validation method that can handle the normal checks for action phase tests and then the additional ones for setup phase tests as well.

Reply:
hell yeah i'll write some up and you can check if they're okay cool?

Reply:
Yup sounds good!